### PR TITLE
update description of `test_order` [ci skip]

### PR DESCRIPTION
**Pull request by y-yagi**
Follow up to 5f777e4b5ee2e3e8e6fd0e2a208ec2a4d25a960d

### CODE REVIEW

File: guides/source/configuring.md [hunk 1]
```diff
@@ -592,7 +592,7 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.bare` enables or disables the loading of `active_support/all` when booting Rails. Defaults to `nil`, which means `active_support/all` is loaded.
 
-* `config.active_support.test_order` sets the order that test cases are executed. Possible values are `:random` and `:sorted`. This option is set to `:random` in `config/environments/test.rb` in newly-generated applications. If you have an application that does not specify a `test_order`, it will default to `:sorted`, *until* Rails 5.0, when the default will become `:random`.
+* `config.active_support.test_order` sets the order that test cases are executed. Possible values are `:random` and `:sorted`. Defaults to `:random`.
```
Comment (vipulnsward) [inline guides/source/configuring.md]: `sets the order in which the test the cases are executed`

Comment (y-yagi) [inline guides/source/configuring.md]: Thanks! Updated.

File: guides/source/configuring.md [hunk 2]
```diff
@@ -592,7 +592,7 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.bare` enables or disables the loading of `active_support/all` when booting Rails. Defaults to `nil`, which means `active_support/all` is loaded.
 
-* `config.active_support.test_order` sets the order that test cases are executed. Possible values are `:random` and `:sorted`. This option is set to `:random` in `config/environments/test.rb` in newly-generated applications. If you have an application that does not specify a `test_order`, it will default to `:sorted`, *until* Rails 5.0, when the default will become `:random`.
+* `config.active_support.test_order` sets the order in which the test the cases are executed. Possible values are `:random` and `:sorted`. Defaults to `:random`.
```
Comment (vipulnsward) [inline guides/source/configuring.md]: @y-yagi you missed an extra "the" here 😄

Comment (y-yagi) [inline guides/source/configuring.md]: Oops :sweat: Thanks! I updated again.

**Comment by vipulnsward** [comment]
Thanks!

**Pull request merged**